tests/e2e/screens/test_single_screens.py

```diff
@@ -87,6 +87,21 @@ def test_check_in_screen(
 
         row.click()
         modal = lan_page.locator('.modal-dialog')
+        # The modal should not be visible since check-in is not open yet
+        expect(modal).not_to_be_visible()
```

Comment:
This doesn't test that the modal won't open unfortunately.  It tests that it isn't open in the instant after you click, but it takes time to open the modal, it might open after this test passes.

In test_spectator_role I had to do this:

```
        # We should NOT see the modal
        row = rows.filter(has_text='ALYX')
        row.click()
        modal = auth_page.locator('.modal-dialog')
        auth_page.wait_for_timeout(200)
        expect(modal).not_to_be_visible()
```

It's not nice, but I don't have a better solution for the moment